content/zh-cn/docs/concepts/architecture/_index.md

```diff
@@ -147,16 +147,6 @@ Node components run on every node, maintaining running pods and providing the Ku
 
 {{< glossary_definition term_id="kubelet" length="all" >}}
 
-<!--
-### kube-proxy (optional) {#kube-proxy}
-
-{{< glossary_definition term_id="kube-proxy" length="all" >}}
```

Comment:
Only need to remove the line 153:

```
{{< glossary_definition term_id="kube-proxy" length="all" >}}
```

How about to have a try?

Reply:
Thanks, updated!